### DISCUSSION TITLE
Welcome to PyOptInterface Discussions!

### DISCUSSION THREAD

**Discussion by metab0t** [metab0t/PyOptInterface#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by huiq666** [comment]


**Reply by metab0t** [reply]
Thanks for your interests!

General constraint of Gurobi is not supported by PyOptInterface now. Of course, we can implement similar APIs like gurobipy to support adding general constraint without too much effort. Can you describe on your usage of general constraints and your expectation for its implementation in PyOptInterface briefly? 

By the way, please open a new thread to discuss a specific topic next time. Thanks.
